torch/csrc/jit/codegen/cuda/lower_index.h

```diff
@@ -65,7 +66,10 @@ class TORCH_CUDA_CU_API IndexLowering : private OptOutConstDispatch {
 
   void generate(const std::vector<Expr*>& exprs);
 
-  Val* lowerSrcIndex(Val* val, Val* dst) const;
+  Val* lowerSrcIndex(
```

Comment:
Does the source IDs of the map need to be concrete? Please add a comment about it.

Reply:
added